Configuration/AlCa/python/autoCond.py

```diff
@@ -16,9 +16,9 @@
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
     'run2_design'       :   '110X_mcRun2_design_v3',
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
-    'run2_mc'           :   '110X_mcRun2_asymptotic_v3',
+    'run2_mc'           :   '110X_mcRun2_asymptotic_Candidate_2019_11_05_13_17_35',
```

Comment:
@christopheralanwest , shall we have proper GT tag name instead?

Reply:
Here are new Global Tags which can be used instead of the candidates:

110X_mcRun2_asymptotic_v4 (replaces110X_mcRun2_asymptotic_Candidate_2019_11_05_13_17_35)

and

110X_mcRun2cosmics_startup_deco_v3 (replaces 110X_mcRun2cosmics_startup_deco_Candidate_2019_11_05_13_38_13)

diffs are here:
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/110X_mcRun2_asymptotic_v4/110X_mcRun2_asymptotic_Candidate_2019_11_05_13_17_35
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/110X_mcRun2cosmics_startup_deco_v3/110X_mcRun2cosmics_startup_deco_Candidate_2019_11_05_13_38_13

Reply:
Thanks, I just pushed a commit to update them in autoCond.py